packages/@react-aria/overlays/src/useOverlay.ts

```diff
@@ -49,7 +49,12 @@ interface OverlayAria {
   overlayProps: HTMLAttributes<HTMLElement>
 }
 
-const visibleOverlays: RefObject<HTMLElement>[] = [];
+interface OpenOverlay {
+  ref: RefObject<HTMLElement>,
+  onClose: () => void
+}
+
+export const visibleOverlays: OpenOverlay[] = [];
```

Comment:
We should make sure that this isn't exported from the package (e.g. `index.ts`).